notebooks/community/model_garden/model_garden_pytorch_detectron2.ipynb

```diff
@@ -0,0 +1,1237 @@
+{
```

Comment:
<p>the Endpoint -&gt; an Endpoint resource</p><p>GCS -&gt; Cloud Storage</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1783/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Done.